Error: Cannot find module 'generate-defaults'

```
npm i -g generate generate-swap-project@latest
/usr/local/bin/gen -> /usr/local/lib/node_modules/generate/bin/generate.js
/usr/local/bin/generate -> /usr/local/lib/node_modules/generate/bin/generate.js
+ generate@0.14.0
+ generate-swap-project@0.4.1
added 20 packages and updated 4 packages in 17.216s
root@966ca5819962:~# ls
root@966ca5819962:~# gen swap-project
[07:18:00] starting generate
[07:18:01] ✔ running tasks: [ 'swap-project' ]
module.js:544
    throw err;
    ^

Error: Cannot find module 'generate-defaults'
    at Function.Module._resolveFilename (module.js:542:15)
    at Function.Module._load (module.js:472:25)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at Generate.exports.default (/usr/local/lib/node_modules/generate-swap-project/dist/lib/generator.js:19:11)
    at Env.invoke (/usr/local/lib/node_modules/generate/node_modules/base-env/lib/env.js:118:23)
    at Generate.<anonymous> (/usr/local/lib/node_modules/generate/node_modules/base-generators/lib/generator.js:62:30)
    at Generate.getGenerator (/usr/local/lib/node_modules/generate/node_modules/base-generators/index.js:157:22)
    at Object.exports.process (/usr/local/lib/node_modules/generate/node_modules/base-generators/lib/tasks.js:127:24)
    at /usr/local/lib/node_modules/generate/node_modules/base-generators/lib/tasks.js:86:23
    at Array.reduce (<anonymous>)
    at exports.parse (/usr/local/lib/node_modules/generate/node_modules/base-generators/lib/tasks.js:85:15)
    at Generate.generate (/usr/local/lib/node_modules/generate/node_modules/base-generators/index.js:490:21)
    at /usr/local/lib/node_modules/generate/node_modules/base-generators/index.js:475:18
    at iterate (/usr/local/lib/node_modules/generate/node_modules/async-each-series/index.js:8:5)
    at Function.module.exports (/usr/local/lib/node_modules/generate/node_modules/async-each-series/index.js:20:3)
root@966ca5819962:~# 

```

--- a/package.json
+++ b/package.json
@@ -67,7 +67,6 @@
     "chalk": "^2.3.0",
     "delete": "^1.1.0",
     "generate": "^0.14.0",
-    "generate-dest": "^0.1.4",
     "gulp": "^3.9.1",
     "gulp-bump": "^3.0.0",
     "gulp-cli": "^2.0.0",
@@ -83,6 +82,8 @@
     "babel-runtime": "^6.26.0",
     "debug": "^3.1.0",
     "extend": "^3.0.1",
+    "generate-defaults": "^0.6.6",
+    "generate-dest": "^0.1.4",
     "is-valid-app": "^0.3.0",
     "prettier-inspect": "^1.0.0"
   },

--- a/src/lib/generator.js
+++ b/src/lib/generator.js
@@ -3,6 +3,9 @@ import extend from 'extend'
 import path from 'path'
 import Logger from './utils/Logger'
 
+import generateDefaults from 'generate-defaults'
+import generateDest from 'generate-dest'
+
 const log = new Logger('generate-swap-project')
 
 export default function (app) {
@@ -17,12 +20,12 @@ export default function (app) {
    * Plugins
    */
 
-  app.use(require('generate-defaults'))
+  app.use(generateDefaults)
 
   /**
    * Micro generators (as plugins)
    */
-  app.register('destination-directory', require('generate-dest'))
+  app.register('destination-directory', generateDest)
 
   /**
    * Scaffold out a(n) swap-project project. Also aliased as the [default](#default) task.

